docs/en/ingest-management/release-notes/release-notes-8.11.asciidoc

```diff
@@ -0,0 +1,225 @@
+For more information, refer to {agent-pull}3593[#3593].
+====
+
+[discrete]
+[[new-features-8.11.0]]
+=== New features
+
+The 8.11.0 release Added the following new and notable features.
+
+{fleet-server}::
+* Add new `upgrade_details` attribute to checkin requests and the agent index. {fleet-server-pull}2901[#2901]
+
+{agent}::
+* Add support for processors in hints-based Kubernetes autodiscover. {agent-pull}3107[#3107] {agent-issue}2959[#2959]
+* Print out {agent} installation steps to show progress. {agent-pull}3338[#3338]
+* Add colors to {agent} log messages based on their level. {agent-pull}3345[#3345]
+* Improve {agent} uninstall on Windows by adding delay between retries when file removal is blocked by busy files. {agent-pull}3431[#3431] {agent-issue}3221[#3221]
+
+[discrete]
+[[enhancements-8.11.0]]
+=== Enhancements
+
+{fleet-server}::
+* Expand APM traces to track coordinator and monitor transactions. Add additonal spans across all API endpoints to better track what the server does. Add spans to bulker interactions that link with the queue flush transaction that the bulk action is executed through. {fleet-server-pull}2929[#2929]
+* Add endpoint to serve PGP keys that clients can use when validating upgrades in cases where the embedded PGP key in a client is compromised and the client can't reach the internet. {fleet-server-pull}2977[#2977] {fleet-server-issue}2887[#2887]
+* Add ActionLimit and a Gzip writer pool to handle checkin responses, to help prevent OOM errors when updates are issued to many clients. {fleet-server-pull}2929[#2994]
+* Send errors in API calls and bulker flushes to APM. fleet-server-pull}3053[#3053]
+
+{agent}::
+* Improve {agent} uninstall on Windows by adding delay between retries when file removal is blocked by busy files. {agent-pull}3431[#3431] {agent-issue}3221[#3221]
+* Updating Elastic manifests with NETINFO variable. Setting a new environmental variable `ELASTIC_NETINFO=false` globally disables the `netinfo.enabled` parameter of the `add_host_metadata` processor. This disables the indexing of `host.ip` and `host.mac` fields. {agent-pull}3354[#3354]
+* The {agent} uninstall process now finds and kills any running Watcher process. {agent-pull}3384[#3384] {agent-issue}3371[#3371]
+* Fix the Kubernetes `deploy/kubernetes/creator_k8.sh` script to correcly exclude configmaps. {agent-pull}3396[#3396]
+* Add a secondary fallback for package signature verification, to enable reliable upgrades in air gapped environments where {fleet-server} is the only reachable URI. {agent-pull}3543[#3543] {agent-issue}3264[#3264]
+* Enable tamper protection feature flag by default for {agent} version 8.11.0. {agent-pull}3478[#3478]
+* Increase {agent} monitoring metrics interval from 10s to 60s to reduce load. {agent-pull}3578[#3578]
+* Updated Go version to 1.20.10. {agent-pull}3[#3601]
```

Comment:
We could move this under the Security Updates section since it is primarily to fix CVEs.